awx/main/constants.py

```diff
@@ -16,10 +15,15 @@
     'STANDARD_INVENTORY_UPDATE_ENV',
 ]
 
-plugin_names = tuple(InventorySourceOptions.injectors.keys())
+
+def load_plugin_names():
+    from awx.main.models.inventory import InventorySourceOptions
+
+    plugin_names = tuple(InventorySourceOptions.injectors.keys())
+    return plugin_names
 
 
-CLOUD_PROVIDERS = plugin_names
+CLOUD_PROVIDERS = load_plugin_names
```

Comment:
Perhaps, using graphviz/pylint+pyreverse/`python -Ximporttime` could help you identify the import order and handle that better... https://stackoverflow.com/a/64068258/595220 / https://mg.pov.lt/objgraph/

Reply:
Thank you for making me aware of this tool!

Reply:
You're welcome :) Please feel free to mark this as resolved.